analyses/focal-cn-file-preparation/run-prepare-cn-OpenTarget.sh

```diff
@@ -52,29 +52,34 @@ if [ "$RUN_ORIGINAL" -gt "0" ]; then
     --gtf_file $gtf_file \
     --metadata $histologies_file \
     --filename_lead "cnvkit_annotated_cn" \
-    --seg
+    --seg \
+    --runWXSonly
 
   # Run annotation step for ControlFreeC
   Rscript --vanilla 04-prepare-cn-file.R \
     --cnv_file ${data_dir}/cnv-controlfreec.tsv.gz \
     --gtf_file $gtf_file \
     --metadata $histologies_file \
     --filename_lead "controlfreec_annotated_cn" \
-    --controlfreec
+    --controlfreec \
+    --runWXSonly
 
-  filenameLead=("cnvkit_annotated_cn" "controlfreec_annotated_cn")
+  filenameLead=("cnvkit_annotated_cn" "controlfreec_annotated_cn" "cnvkit_annotated_cn_wxs" "controlfreec_annotated_cn_wxs")
+  chromosomeType=("autosomes" "x_and_y")
   for filename in ${filenameLead[@]}; do
-    for strategy in ${libraryStrategies[@]}; do
-      for chromosome_type in ${chromosomesType[@]}; do
+    for chromosome_type in ${chromosomesType[@]}; do
         Rscript --vanilla rna-expression-validation.R \
           --annotated_cnv_file results/${filename}_${chromosome_type}.tsv.gz \
-          --expression_file ${data_dir}/gene-expression-rsem-fpkm-collapsed.${strategy}.rds \
+          --expression_file ${data_dir}/gene-expression-rsem-tpm-collapsed.rds \
           --independent_specimens_file $independent_specimens_file \
           --metadata $histologies_file \
           --goi_list $goi_file \
-          --filename_lead ${filename}_${chromosome_type}_${strategy}
+          --runWXSonly \
+          --filename_lead ${filename}_${runWXSOnly}_${chromosome_type}
```

Comment:
should this have ${runWXSOnly in the filename here?

Reply:
You are right! Will fix that!